### PR TITLE
Add SAP HANA Manual config to AgnosticD

### PR DESCRIPTION
**Pull request by makentenza**
##### SUMMARY
Add new configuration to AgnosticD. This configuration will be used for 'Techincal Enablement on how to insall SAP HANA on an standalone server'

##### COMPONENT NAME
Config 'sap-hana-manual'

##### ADDITIONAL INFORMATION
This has been tested in the RED Cluster. Additional modifications may be required to accommodate BLUE configuration once existing blockers (RHEL 8 images, RHEL 8 Subscriptions) are removed, but this is ready to go.

### CODE REVIEW

File: ansible/configs/sap-hana-manual/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (stencell) [inline ansible/configs/sap-hana-manual/requirements.yml]: If this isn't used, can you just delete the file?

Comment (makentenza) [reply ansible/configs/sap-hana-manual/requirements.yml]: Done

**Comment by fridim** [comment]
duplicate of #962 ? can you close on of them ?

**Comment by makentenza** [comment]
> duplicate of #962 ? can you close on of them ?

These are 2 different config @fridim, one for manual deployment of HANA and the second one for automated deployment using Ansible. I could try to merge both and use variables to decide to install Tower and configure Ansible or not, is that what you mean?

**Pull request closed without merging**